deploy/kubernetes/rbac.yaml

```diff
@@ -47,6 +47,10 @@ rules:
   - apiGroups: ["apiextensions.k8s.io"]
     resources: ["customresourcedefinitions"]
     verbs: ["create", "list", "watch", "delete"]
+    # access to leases is required for leader election
```

Comment:
Oh actually, we only need access to the Lease object in the namespace of the sidecar right? not all namespaces in the cluster?

Reply:
yes, just just the current namespace. I was lazy and didn't want to create a dedicated role just for this but let me do that now

Reply:
done